acl.go

```diff
@@ -300,43 +301,44 @@ func (odbi *ovndb) rowToACL(uuid string) *ACL {
 
 // Get all acl by lswitch
 func (odbi *ovndb) aclListImp(lsw string) ([]*ACL, error) {
-	var listACL []*ACL
-
 	odbi.cachemutex.RLock()
 	defer odbi.cachemutex.RUnlock()
 
 	cacheLogicalSwitch, ok := odbi.cache[TableLogicalSwitch]
 	if !ok {
 		return nil, ErrorNotFound
 	}
-	var lsFound bool
 	for _, drows := range cacheLogicalSwitch {
 		if rlsw, ok := drows.Fields["name"].(string); ok && rlsw == lsw {
 			acls := drows.Fields["acls"]
 			if acls != nil {
 				switch acls.(type) {
 				case libovsdb.OvsSet:
 					if as, ok := acls.(libovsdb.OvsSet); ok {
-						for _, a := range as.GoSet {
+						listACL := make([]*ACL, len(as.GoSet))
+						for i, a := range as.GoSet {
 							if va, ok := a.(libovsdb.UUID); ok {
 								ta := odbi.rowToACL(va.GoUUID)
-								listACL = append(listACL, ta)
+								listACL[i] = ta
+							} else {
```

Comment:
@cathy-zhou  Also, one more thing. A separate PR/commit for error handling would be preferred. Hence, this patch will be just to initialize slices with proper mem allocations

Reply:
sure, thanks for your comments. I just repushed the updated code.